.pre-commit-config.yaml

```diff
@@ -7,7 +7,10 @@ repos:
       - id: end-of-file-fixer
         types: [python]
       - id: trailing-whitespace
-      - id: requirements-txt-fixer
+        exclude: |
+          (?x)^(
+            data_structures/heap/binomial_heap.py
+          )$
```

Comment:
This got removed here: https://github.com/TheAlgorithms/Python/commit/57ccabbaeb0f32165271e3a218bc9c6dcfc21823. I don't know how it got added back here?